### ISSUE TITLE
Unable to craft MPSA modules. Get "Server has disallowed this module. Sorry!"

### ISSUE THREAD

**Issue by phishphan420-zz** [Eximius88/ModularPowersuitsAddons#3]
We are having issues on our server with players not being able to craft any of the MPSA modules. Everytime we try to install one of the modules such as the thermal or kinetic generator we get a message saying "Server has disallowed this module. Sorry!" The odd thing is that the modules all work totally fine in SSP but not in SMP. I don't understand why they will not work as I checked the configs against the current DW20 pack and were using the exact same configs. Wondering if this is a bug with MPSA or if I'm just missing something much more obvious. Any help would be greatly appreciated.

Thanks,

Phish (Matt)

**Comment by Eximius88** [comment]
What version of MPSA, MPS, and Numina are you using?

**Comment by phishphan420-zz** [comment]
MPS - 0.10.0-92
MPSA - build 884
Numina - 0.1.2-55

Configs -
MPS - http://pastebin.com/3er0cdBW
MPSA - http://pastebin.com/c6kBs5ak
Numina - http://pastebin.com/Fgz814p0

**Comment by Eximius88** [comment]
Odd. That was a bug in earlier builds but I released 884 to specifically fix it. I will look into it asap, but I had a hard drive issue and lost some work that I hadn't commited to the repo yet so I am still recovering.

**Comment by phishphan420-zz** [comment]
Thanks a ton man. I saw on your post about losing the work. Major bummer. Same thing happened to me recently with a whole RAID 5 array going down. Not fun. Good luck fixing it. Whenever you get to it would be much appreciated. =]

**Comment by phishphan420-zz** [comment]
Hey so I solved the problem. This is going to sounds absolutely ridiculous and stupid of me but it seems that when we started experiencing the issue one of our other admins whose been gone this week FTP'd into the server and disabled MPSA to prevent any further issues. In checking all of the configs I never thought to check the actual JAR file. As soon as I re-enabled the mod everything worked again (as expected). Sorry for the confusion and making you nuts about a bug you already fixed. LOVE the mod and keep up the good work!
